README.md

```diff
@@ -75,37 +75,44 @@ Usage: ruby ticketmatch.rb [options]
     -h, --help                       this message
 ```
 
-# Batch processing
+# Batch processing for puppet-agent and its components
 
-If you need to check all the components that comprise `puppet-agent` for a release, use
-`pa_matchbatch.sh`. It requires one argument: the revision (SHA or reference) of `puppet-agent` you
-care about. It will process `puppet-agent` and all its components that are tagged with a SHA.
 
-Component directories will be cloned from the path provided in their `component.json` file and the
-appropriate branch set. If a given component already exists, a Git fetch will be performed to get
-things up to date (see environment variables, below, for additional options).
+The `pa_matchbatch.sh` script clones a revision of puppet-agent and all of its components, then runs
+ticketmatch on everything; This is intended for use during puppet-agent releases.
 
-## How to run pa_matchbatch (one way)
```

Comment:
I didn't understand why there were two ways described here, so I replaced this section with one way